### PR TITLE
Revamp 2ndline dashboard post-Replatforming

### PR DESCRIPTION
**Pull request by ChrisBAshton**
## What & Why

This PR pulls in relevant code from [govuk-secondline-blinken](https://github.com/alphagov/govuk-secondline-blinken) (for Icinga alerts information) and [frame-splits](https://github.com/alphagov/frame-splits) (for owning its own layout), to make the 2nd line dashboard useful again. Once this PR is merged, *both of those repos can be archived*. 🎉 

Once this is merged, https://govuk-zendesk-display-screen.herokuapp.com/ will automatically redirect to `https://govuk-zendesk-display-screen.herokuapp.com/frame-splits?<lots of parameters>` and render a dashboard that is relevant in the world of Replatforming. It has three sections:

1. A summary of resources (as documentation is currently somewhat sporadic, so it would be useful for 2nd liners to have this consolidated and easily reachable from their dashboard).
2. Zendesk ticket counts (including, for the first time, 'Low Priority' ticket counts. These tend to be in a 'Pending' state so the count should generally be low, and this serves as a reminder to check those tickets where they often get forgotten. We can always hide the low priority ones again if needed.
3. Icinga alert counts (Icinga alerts can largely be ignored now, but with some exceptions - documented in the summary frame - so this still serves as a handy shortcut to the Icinga environments, and also highlights any alerts that haven't been ACK'd).

## Screenshots

Before:

> 
Screenshot 2023-04-17 at 15 40 59

After:

> 
layout

### CODE REVIEW

File: public/assets/js/blinken-config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var blinken_config = {
+  "groups": [
+    {
+      "id": "govuk",
+      "name": "",
+      "environments": [
+        {
+          "name": "Production (AWS)",
```
Comment (MuriloDalRi) [inline public/assets/js/blinken-config.js:8]: We've been on AWS for ages now, do we still need these labels?

**Review by MuriloDalRi — APPROVED** [review]

**Comment by MuriloDalRi** [comment]
Nice!

**Review by beccapearce — APPROVED** [review]
Looks good!

File: public/assets/js/blinken-config.js (hunk 1 above)
Comment (sengi) [reply public/assets/js/blinken-config.js:8]: Maybe change them to EC2 or something to indicate that they're now legacy? (until we remove Icinga altogether, which will be soon but we're not quite there yet)

File: views/summary.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <% else %>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <% end %>
+    <link rel="stylesheet" href="/assets/css/zendesk.css" crossorigin="anonymous">
+    <title>Summary</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>GOV>UK Technical 2nd line dashboard</h1>
```
Comment (brucebolt) [inline views/summary.erb:18]: There's a typo here
```suggestion
        <h1>GOV.UK Technical 2nd line dashboard</h1>
```

**Review by brucebolt — APPROVED** [review]
Looks good to me 👍.  But spotted a typo in one of the views.

**Pull request merged**